.eslintrc

```diff
@@ -13,7 +13,8 @@
     },
     "rules": {
       "import/no-extraneous-dependencies": ["off"],
-      "import/no-unresolved": ["off"]
+      "import/no-unresolved": ["off"],
+      "react/jsx-filename-extension": ["off"]
```

Comment:
too bad there are no trailing commas in json ;)

Reply:
Yes it's a pity,